lib/jobs/create-default-pollers.js

```diff
@@ -113,7 +113,32 @@ function createDefaultPollersJobFactory(
         return Promise.map(nodes, function(nodeId) {
             return self._createWorkitem(nodeId, pollerTemplate, "redfish-obm-service");
         });
-    }; 
+    };
+
+    CreateDefaultPollersJob.prototype.createIpmiOrSnmpPoller = function(nodes, pollerTemplate, sourceQuery, sourceType) {
```

Comment:
There isn't IPMI or SNMP handling in this function, why call it createIpmiOrSnmpPoller, I think it could be refined

Reply:
Because IPMI and SNMP share the same create process, the only difference is the error message. Another reason I extract this function is to align with Line 85, 95 and 111